lib/application.js

```diff
@@ -202,23 +202,39 @@ class Application extends api.events.EventEmitter {
     });
   }
 
+  loadSchemas(callback) {
+    const dir = api.path.join(this.dir, 'schemas');
+    const loaderPath = api.path.join(dir, 'config.js');
+    impress.createScript(this, loaderPath, (err, getMsConfig) => {
+      if (err) {
+        callback(err);
+        return;
+      }
+
+      const { options, config } = getMsConfig();
+      options.context = this;
+
+      api.metaschema.fs.load(dir, options, config).then(([errors, schema]) => {
+        if (errors.length) {
+          console.error(errors);
```

Comment:
Use `impress.log.error()`. I think we can use `console.*()` only before `metalog` initialization.